openupgradelib/openupgrade_160.py

```diff
@@ -389,7 +389,7 @@ def _convert_field_bootstrap_4to5_sql(cr, table, field, ids=None):
     params = ()
     if ids:
         sql += "WHERE id IN %s"
-        params = (ids,)
+        params = (*ids,)
```

Comment:
Incorrect patch

```suggestion
        params = (tuple(ids),)
```

Reply:
Right, I would have expected to work both ways but it doesn't